token/filetoken/filetoken.go

```diff
@@ -0,0 +1,119 @@
+package filetoken
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/puppetlabs/pe-sdk-go/log"
+	"github.com/puppetlabs/pe-sdk-go/token"
+)
+
+// fileToken struct
+type fileToken struct {
+	path string
+}
+
+// NewFileToken constructs a new filetoken
+func NewFileToken(path string) token.Token {
+	fileToken := fileToken{path: path}
+	return &fileToken
+}
+
+// Read reads the token from a file and returns a valid token.
+// If the token is not valid an error will be return
+func (ft *fileToken) ReadTokenFile() (string, error) {
+	var err error
+
+	tokenPath, err := getPath(&ft.path)
+	if err != nil {
+		return "", err
+	}
+
+	data, err := ioutil.ReadFile(*tokenPath)
+	if err != nil {
+		return "", err
+	}
+
+	token := strings.TrimRight(string(data), "\r\n")
+
+	valid := isValid(token)
+	if !valid {
+		err = fmt.Errorf("Token %s is invalid", token)
+		return "", err
+	}
+
+	return token, nil
+}
+func (ft *fileToken) DeleteTokenFile() (string, error) {
```

Comment:
It is needed that this method returns a string? Also, method could be named just `DeleteFile`

Reply:
I considered that the output should contain a message that says the file has been deleted, but also show the actual token that has been deleted. But this is not the case in the previous version of puppet-access.

Reply:
We should keep, whenever possible, same output like previous version